pymeasure/instruments/keithley/keithley2182.py

```diff
@@ -0,0 +1,409 @@
+        Slow respectively. """,
+        validator=truncated_range,
+        values=[0.01, 60],
+        dynamic=True,
+    )
+    temperature_reference_junction = Instrument.control(
+        ":SENS:TEMP:RJUN:RSEL?", ":SENS:TEMP:RJUN:RSEL %s",
+        """ A property that controls the type of thermocouple reference
+        junction. Default is internal.""",
+        validator=strict_discrete_set,
+        values=['SIM', 'INT'],
+    )
+    temperature_simulated_reference = Instrument.control(
+        ":SENS:TEMP:RJUN:SIM?", ":SENS:TEMP:RJUN:SIM %g",
+        """ A property that controls the value of the simulated thermocouple
+        reference junction in Celsius. Default is 23 C.""",
+        validator=truncated_range,
+        values=[0, 60],
+    )
+    internal_temperature = Instrument.measurement(
+        ":SENS:TEMP:RTEM?",
+        """ Query the internal temperature in Celsius."""
+    )
+
+    ##############
+    # Statistics #
+    ##############
+
+    buffer_points = Instrument.control(
+        ":TRAC:POIN?", ":TRAC:POIN %d",
+        """ An integer property that controls the number of buffer points. This
+        does not represent actual points in the buffer, but the configuration
+        value instead. """,
+        validator=truncated_range,
+        values=[2, 1024],
+        cast=int
+    )
+    mean = Instrument.measurement(
+        ":CALC2:FORM MEAN;:CALC2:STAT ON;:CALC2:IMM?;",
+        """ Returns the calculated mean (average) from the buffer data. """
+    )
+    maximum = Instrument.measurement(
+        ":CALC2:FORM MAX;:CALC2:STAT ON;:CALC2:IMM?;",
+        """ Returns the calculated maximum from the buffer data. """
+    )
+    minimum = Instrument.measurement(
+        ":CALC2:FORM MIN;:CALC2:STAT ON;:CALC2:IMM?;",
+        """ Returns the calculated minimum from the buffer data. """
+    )
+    standard_dev = Instrument.measurement(
+        ":CALC2:FORM SDEV;:CALC2:STAT ON;:CALC2:IMM?;",
+        """ Returns the calculated standard deviation from the buffer data. """
+    )
+
+    ###########
+    # Trigger #
+    ###########
+
+    trigger_count = Instrument.control(
+        ":TRIG:COUN?", ":TRIG:COUN %d",
+        """ An integer property that controls the trigger count,
+        which can take values from 1 to 9,999. """,
+        validator=truncated_range,
+        values=[1, 9999],
+        cast=int
+    )
+    trigger_delay = Instrument.control(
+        ":TRIG:DEL?", ":TRIG:DEL %g",
+        """ A floating point property that controls the trigger delay
+        in seconds, which can take values from 0 to 999999.999 s. """,
+        validator=truncated_range,
+        values=[0, 999999.999]
+    )
+
+    ###########
+    # Methods #
+    ###########
+
+    def __init__(self, adapter, name="Keithley 2182 Nanovoltmeter",
+                 read_termination='\r', **kwargs):
+        super().__init__(
+            adapter, name, **kwargs
+        )
```

Comment:
Please add, whether it supports SCPI or not, for example if it does:
```suggestion
        super().__init__(
            adapter, name, includeSCPI=True, **kwargs
        )
```